### PR TITLE
added Alletra 9k/Primera/3par port requirements

### PR DESCRIPTION
**Pull request by c-snell**
Signed-off-by: Chris Snell <snell.chris@gmail.com>

### CODE REVIEW

File: docs/container_storage_provider/hpe_alletra_9000/index.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ Refer to the HPE Single Point of Connectivity Knowledge (SPOCK) for specific pla
 * [HPE Primera](https://h20272.www2.hpe.com/SPOCK/Pages/spock2Html.aspx?htmlFile=hw_primera.html)
 * [HPE 3PAR](https://h20272.www2.hpe.com/SPOCK/Pages/spock2Html.aspx?htmlFile=hw_3par.html)
 
+### Network Port Requirements
+
+The HPE Alletra 9000, Primera and 3PAR Container Storage Provider requires the following ports to be open between the array and the Kubernetes cluster running the HPE CSI Driver for Kubernetes.
```
Comment (datamattsson) [inline docs/container_storage_provider/hpe_alletra_9000/index.md:22]: Can we insert "... Kubernetes cluster worker nodes running ..." to clarify we don't care about what the control-plane nodes do.

File: docs/container_storage_provider/hpe_alletra_9000/index.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ Refer to the HPE Single Point of Connectivity Knowledge (SPOCK) for specific pla
...
+
+| Port | Protocol | Description |
+| ---- | -------- | ----------- |
+| 443 | HTTPS | WSAPI (HPE Alletra 9000/Primera) |
+| 8080 | HTTPS | WSAPI (HPE 3PAR) |
```
Comment (datamattsson) [inline docs/container_storage_provider/hpe_alletra_9000/index.md:27]: Is this truly HTTPS on port 8080?

**Review by datamattsson — CHANGES_REQUESTED** [review]
A few questions/additions.

File: docs/container_storage_provider/hpe_alletra_9000/index.md (hunk 2 above)
Comment (c-snell) [reply docs/container_storage_provider/hpe_alletra_9000/index.md:27]: Yes

File: docs/container_storage_provider/hpe_alletra_9000/index.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Refer to the HPE Single Point of Connectivity Knowledge (SPOCK) for specific pla
 
 ### Network Port Requirements
 
-The HPE Alletra 9000, Primera and 3PAR Container Storage Provider requires the following ports to be open between the array and the Kubernetes cluster running the HPE CSI Driver for Kubernetes.
+The HPE Alletra 9000, Primera and 3PAR Container Storage Provider requires the following ports to be open between the array and the Kubernetes cluster worker nodes running the HPE CSI Driver for Kubernetes.
```
Comment (datamattsson) [inline docs/container_storage_provider/hpe_alletra_9000/index.md:22]: I might be splitting hairs here but I think we need to clarify that it's array inbound TCP traffic that we're talking about.

File: docs/container_storage_provider/hpe_alletra_9000/index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Refer to the HPE Single Point of Connectivity Knowledge (SPOCK) for specific pla
...
 
 | Port | Protocol | Description |
 | ---- | -------- | ----------- |
```
Comment (datamattsson) [inline docs/container_storage_provider/hpe_alletra_9000/index.md:25]: For this table to be comprehensive, should we add 3260 for iSCSI too?

| 3260 | iSCSI | Optional, dependent on data protocol being used. |

**Review by datamattsson — CHANGES_REQUESTED** [review]
A few nits.

File: docs/container_storage_provider/hpe_alletra_9000/index.md [hunk 5]
```diff
@@ -17,6 +17,16 @@ Refer to the HPE Single Point of Connectivity Knowledge (SPOCK) for specific pla
 * [HPE Primera](https://h20272.www2.hpe.com/SPOCK/Pages/spock2Html.aspx?htmlFile=hw_primera.html)
 * [HPE 3PAR](https://h20272.www2.hpe.com/SPOCK/Pages/spock2Html.aspx?htmlFile=hw_3par.html)
 
+### Network Port Requirements
+
+The HPE Alletra 9000, Primera and 3PAR Container Storage Provider requires the following TCP ports to be open inbound between the array and the Kubernetes cluster worker nodes running the HPE CSI Driver for Kubernetes.
```
Comment (datamattsson) [inline docs/container_storage_provider/hpe_alletra_9000/index.md:22]: The HPE Alletra 9000, Primera and 3PAR Container Storage Provider requires the following TCP ports to be open inbound **to** the array **from** the Kubernetes cluster worker nodes running the HPE CSI Driver for Kubernetes.

**Review by datamattsson — CHANGES_REQUESTED** [review]
Another clarification + some tutorial stuff got added in this PR.

**Review by datamattsson — APPROVED** [review]
LGTM

**Pull request merged**